libraries/AP_ExternalAHRS/AP_ExternalAHRS.cpp

```diff
@@ -238,7 +238,6 @@ bool AP_ExternalAHRS::pre_arm_check(char *failure_msg, uint8_t failure_msg_len)
 
     if (!state.have_origin) {
         hal.util->snprintf(failure_msg, failure_msg_len, "ExternalAHRS: No origin");
-        return false;
```

Comment:
Doesn't this failure get lost now?

Maybe it would be better to re-order so that the backend pre-arms come before the origin check?

Reply:
> Doesn't this failure get lost now?
> 
> Maybe it would be better to re-order so that the backend pre-arms come before the origin check?

Yea, a re-order would solve it.

Reply:
Yea, totally. I re-ordered it, and it works great now!